stories/examples/react-autosuggest.js

```diff
@@ -15,38 +15,40 @@ class Examples extends Component {
     this.setState({selectedColor})
   }
 
-  onStateChange = changes => {
```

Comment:
Changes here aren't as dramatic as it looks. Just wrapped everything in a `this.setState` callback.